boards/intel/adsp/Kconfig.defconfig

```diff
@@ -8,3 +8,18 @@ config DAI_SSP_HAS_POWER_CONTROL
 	def_bool y
 
 endif
+
+if BOARD_INTEL_ADSP
+
+config MAIN_STACK_SIZE
+	default 2048 if BOARD_INTEL_ADSP_CAVS25
+	default 2048 if BOARD_INTEL_ADSP_CAVS25_TGPH
+	default 2048 if BOARD_INTEL_ADSP_ACE15_MTPM
+	default 2048 if BOARD_INTEL_ADSP_ACE15_MTPM_SIM
+	default 2048 if BOARD_INTEL_ADSP_ACE20_LNL
+	default 2048 if BOARD_INTEL_ADSP_ACE20_LNL_SIM
+	default 4096 if BOARD_INTEL_ADSP_ACE30
+	default 4096 if BOARD_INTEL_ADSP_ACE30_PTL
+	default 4096 if BOARD_INTEL_ADSP_ACE30_PTL_SIM
```

Comment:
ace30_ptl_sim originally had the default value set to 2048, is the change to 4096 intentional?

Reply:
Since MMU is enabled, the stack must be aligned to page size (which is 4096 bytes). So I update the value to avoid confusion.

Reply:
Ok, I understand.